cmd/falco/runner.go

```diff
@@ -51,16 +51,19 @@ type StatsResult struct {
 	Lines       int    `json:"lines"`
 }
 
-type RunMode struct {
-	isMain bool
-	isStat bool
-}
+type RunMode int
+
+const (
+	RunModeLint RunMode = 0x000001
+	RunModeStat RunMode = 0x000010
+)
```

Comment:
Runner no longer needs to recognize the VCL is main or not, so RunMode could be simple as bit mode.